src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -6,19 +6,16 @@
 
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
 import seedu.address.model.tag.Tag;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
     public static Person[] getSamplePersons() {
-        return new Person[] {
+        return new Person[] {};
+        /* return new Person[] {
```

Comment:
Is there a particular reason for keeping this comment block? Should it be deleted instead or replaced with a new sample data?

Reply:
I have removed the commented block, and I believe we do not need the sample data at all. I will look into it in the future to see if we can remove this class altogether. Thanks.